### PR TITLE
Remove Tox, use podman in conftest to run selenium webdriver

### PR DESCRIPTION
**Pull request by mshriver**
_Definitely_ a better option than trying to get tox-docker to work in #184 

Thanks @quarckster for the recommendation to run podman within conftest instead of trying to use service container.

### CODE REVIEW

**Review by quarckster — APPROVED** [review]

File: .github/workflows/test_suite.yml [hunk 1]
```diff
@@ -37,36 +37,67 @@ jobs:
...
         browser: [ chrome, firefox ]
         python-version: [ pypy3, 3.6, 3.7, 3.8 ]
 
+    services:
+      webdriver:
+        image: quay.io/redhatqe/selenium-standalone:latest
+        ports:
+          - 4444:4444
+          - 8080:8080
+        options: "--shm-size=2g"
```
Comment (quarckster) [inline .github/workflows/test_suite.yml:51]: I believe tests failed because docker ran the container without `--network host`

**Review by quarckster — CHANGES_REQUESTED** [review]
Please see my comment.

File: testing/conftest.py [hunk 2]
```diff
@@ -1,89 +1,158 @@
...
+    wait_for(urlopen, func_args=[selenium_url], timeout=180, handle_exception=True)
+    if browser_name == "firefox":
+        desired_capabilities = DesiredCapabilities.FIREFOX
+    else:
+        desired_capabilities = DesiredCapabilities.CHROME.copy()
+        desired_capabilities["chromeOptions"] = {
+            "args": [
+                "no-sandbox",
+                "disable-infobars"
+                "disable-dev-shm-usage",
```
Comment (quarckster) [inline testing/conftest.py:89]: Should it be `--disable-dev-shm-usage`?

**Review by quarckster — CHANGES_REQUESTED** [review]

**Comment by quarckster** [comment]
@mshriver I've finally fixed all issues in this PR including docs.

**Review by quarckster — APPROVED** [review]

**Comment by mshriver** [comment]
All credit to @quarckster here, thanks for resolving this!

**Pull request merged**